spec/models/forms/condition_computer_spec.rb

```diff
@@ -151,7 +153,7 @@
       #   Q6.3.1                    SR5
       #   Q6.3.2                            SR7
       # Q7                                  SR7
-      # Q8          (nothing because it's a hidden question)
+      # Q8          (nothing because it's a disabled question)
```

Comment:
Is this true or should this stay `hidden` and you add another line to the comment for `Q9` which is `disabled`?

Reply:
Yep! Silly mistake. I've fixed this but am not sure if the passing spec is testing the correct behavior... I haven't explicitly added a skip rule to Q8 but it ends up being associated with SR7, is that because it's the "end" that SR7 is skipping to? I don't think I understand this spec very well.

```
      # Q8          SR7 (hidden question)
      # Q9          (nothing because it's a disabled question)
```